packages/react-scripts/config/webpackDevServer.config.js

```diff
@@ -72,6 +72,10 @@ module.exports = function(proxy, allowedHost) {
     // WebpackDevServer is noisy by default so we emit custom message instead
     // by listening to the compiler events with `compiler.hooks[...].tap` calls above.
     quiet: true,
+    // WebpackDevServer has an option to write every build to disk. This can be
+    // useful for certain use cases such as developing browser extensions where one
+    // would need to otherwise do full builds.
+    writeToDisk: false || process.env.WRITE_TO_DISK === 'true',
```

Comment:
This isn't really comment on the intent of the PR, but I think `false || <expression>` is always exactly equivalent to `<expression>` in JavaScript.

```suggestion
    writeToDisk: process.env.WRITE_TO_DISK === 'true',
```

Reply:
Nice catch. I reckon that's left over from some manual testing I did with it.